### DISCUSSION TITLE
Problem with Sergen Codegenerator after Update - Missing Path

### DISCUSSION THREAD

**Discussion by TreasureM** [serenity-is/Serenity#5339] [Help] [answered]
Hello Community,

I am quite new to Serenity and have encountered a significant problem.
I installed Serenity accoring to the documentation found here (https://serenity.is/docs/getting_started/installing_serene_from_visual_studio_gallery) and started with the Movie-Tutorial (https://serenity.is/docs/tutorials/movies/movies).
Everything went without problem till I came to the issue of updating Serenity-Packages (https://serenity.is/docs/tutorials/movies/updating_serenity_packages) which I did by strictly following the instruction. However in the next tutorial chapter another code generation for a table was necessary and this time I encountered the following error:

`Unhandled exception. System.ArgumentException: Path cannot be the empty string or all whitespace. (Parameter 'path')
   at System.IO.Directory.CreateDirectory(String path)
   at Serenity.CodeGenerator.EntityCodeGenerator.Run() in C:\Sandbox\stargen\Serenity\src\Serenity.Net.CodeGenerator\Generator\EntityCodeGenerator.cs:line 47
   at Serenity.CodeGenerator.GenerateCommand.Run(String csproj, String[] args) in C:\Sandbox\stargen\Serenity\src\Serenity.Net.CodeGenerator\Commands\GenerateCommand.cs:line 410
   at Serenity.CodeGenerator.Program.Main(String[] args) in C:\Sandbox\stargen\Serenity\src\Serenity.Net.CodeGenerator\Program.cs:line 103`

The very same error message now comes when I try to generate code via **dotnet sergen g** for any table. Does anyone have any idea how this came to pass and how it can be corrected? Thank you very much.

Sergen is now version 5.0.20.1. Unfortunately I dont know which version it was before updating.

**Comment by VictorTomaili** [comment, marked as answer]
Related to https://github.com/serenity-is/Serenity/issues/5335#issuecomment-755375933 
update your sergen and try again. It's fixed already. 👍

**Reply by prasadaxiom** [reply]
I have tried all the possible ways to update it, haven't succeeded.. Kindly give the steps to update sergen will help. Thank you..
